.storybook/preview.js

```diff
@@ -1,3 +1,16 @@
+import { addDecorator } from '@storybook/react';
+import { ThemeProvider } from 'styled-components';
+import theme from 'styles/theme';
+
+import 'babel-polyfill';
+import 'normalize.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'noty/lib/noty.css';
+import 'react-datepicker/dist/react-datepicker.css';
+import 'styles/app.sass';
+
 export const parameters = {
   actions: { argTypesRegex: '^on[A-Z].*' },
 };
+
+addDecorator((story) => <ThemeProvider theme={theme}>{story()}</ThemeProvider>);
```

Comment:
Wrap all stories with the theme provider, so every component will have the theme context available.